clang/test/CodeGen/X86/x86-builtins-intrinsic.cl

```diff
@@ -0,0 +1,73 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py
+// RUN:  %clang_cc1 -no-opaque-pointers %s -triple x86_64-pc-linux  \
+// RUN:   -emit-llvm  -O0  -x cl -cl-std=CL2.0 -ffake-address-space-map \
```

Comment:
```suggestion
// RUN:   -emit-llvm  -O0 -cl-std=CL2.0 -ffake-address-space-map \
```
Is `-x cl` required? The test already has .cl extension.

Reply:
removed this -x cl option